src/transformers/modeling_utils.py

```diff
@@ -320,9 +320,13 @@ def prune_heads(self, heads_to_prune):
 
         self.base_model._prune_heads(heads_to_prune)
 
-    def save_pretrained(self, save_directory):
+    def save_pretrained(self, save_directory, xla_device=False):
         """ Save a model and its configuration file to a directory, so that it
             can be re-loaded using the `:func:`~transformers.PreTrainedModel.from_pretrained`` class method.
+
+            Arguments:
+                save_directory: directory to which to save.
+                xla_device: True if saving after training on TPU/XLA.
```

Comment:
This makes me a bit uncomfortable. I'm pretty sure the best option would be to save it in the model configuration instead of adding an argument to `save_pretrained`. I'll look into what can be done to have this clean.

Reply:
Your changes have now been merged and I like the way it looks now better 😄